deps/k_release

```diff
@@ -1 +1 @@
-https://github.com/kframework/k/releases/download/v5.0.0-b3f3e28
+https://github.com/kframework/k/releases/download/v5.0.0-80b5cf9
```

Comment:
Did you mean to change the K release?

Reply:
yes, I think I meant to.  But I don't remember exactly why. I think that the tests weren't working as expected without the latest changes in the frontend.